lsp-jedi.el

```diff
@@ -106,23 +105,59 @@ autocompletion performance but loses goto definition."
 (defcustom lsp-jedi-python-library-directories '("/usr/")
   "List of directories which will be considered to be libraries."
   :risky t
-  :type '(repeat string)
-  :group 'lsp-jedi
-  :package-version '(lsp-mode . "6.1"))
+  :type '(repeat directory)
+  :group 'lsp-jedi)
+
+(defcustom lsp-jedi-case-insensitive-completion t
+  "Completions are by default case insensitive."
+  :type 'boolean
+  :group 'lsp-jedi)
+
+(defcustom lsp-jedi-code-action-name-extract-function "jls_extract_def"
+  "Function name generated by the 'extract_function' codeAction."
+  :type 'string
+  :group 'lsp-jedi)
+
+(defcustom lsp-jedi-code-action-name-extract-variable "jls_extract_var"
+  "Variable name generated by the 'extract_variable' codeAction."
+  :type 'string
+  :group 'lsp-jedi)
+
+(defcustom lsp-jedi-completion-resolve-eagerly nil
+  "Return all completion results in initial completion request."
+  :type 'boolean
+  :group 'lsp-jedi)
+
+(defcustom lsp-jedi-workspace-symbols-max-symbols 20
+  "Maximum number of symbols returned by a call to `workspace/symbols'."
+  :type 'integer
+  :group 'lsp-jedi)
+
+(defcustom lsp-jedi-workspace-symbols-ignore-folders
+  [".nox" ".tox" ".venv" "__pycache__" "venv"]
+  "Performance optimization that sets names of folders that are ignored for the workspace symbols action."
+  :type 'lsp-string-vector
+  :group 'lsp-jedi)
 
 (lsp-register-custom-settings
  '(("jedi.enable" lsp-jedi-enable)
    ("jedi.startupMessage" lsp-jedi-startup-message)
    ("jedi.markupKindPreferred" lsp-jedi-markup-kind-preferred)
    ("jedi.trace.server" lsp-jedi-trace-server)
    ("jedi.jediSettings.autoImportModules" lsp-jedi-auto-import-modules)
+   ("jedi.jediSettings.caseInsensitiveCompletion" lsp-jedi-case-insensitive-completion t)
```

Comment:
I think that case insensitive completion must be have default value `nil`

Reply:
It defaults to True on [`jedi-language-server`](https://github.com/pappasam/jedi-language-server/blob/main/jedi_language_server/initialization_options.py#L48) and [coc-jedi](https://github.com/pappasam/coc-jedi/blob/main/package.json#L89).